### ISSUE TITLE
Automatical generator of nut files

### ISSUE THREAD

**Issue by xavitator** [Heijix/nut-lang#15]
It is a generator of nut files based on the nut grammar.
It is writting in Java.
It is updated on [generator_nut branch](https://github.com/Heijix/nut-lang/tree/generator_nut).

**Comment by FauconFan** [comment]
Good idea. Do all your project in a subdirectory in prog_urils, called nut_generator_files for example. ^^

**Comment by xavitator** [comment]
This is the idea, however, i don't see my commit ? i don't know if it is normal ?

**Comment by xavitator** [comment]
ok, it is done.

**Comment by FauconFan** [comment]
Hey, that's great ^^.
Just a few things : 
 - code in english
 - you use **/*.java notation, doesn't work on every system. Use 'find -name "*.java" instead
 - you use hard coded colors... and you use echo. Echo has not the same behaviour in every systems. See what i have written things in the root Makefile.
 - remove the .gitkeep
 - 'lancement jeu' line 46. Copy paste is a bad thing xD
 - you declare FLAG_1 and FLAG_2, put some more explicit names, we can't understand...
 - FLAG_2 is not used, and we don't need a variable for javac options
 - You put a copyright in the makefile, we have no license yet and globally heijix will have a opensource license. You cannot use a copyright...

**Comment by FauconFan** [comment]
The program also crash if we don't call the program with the good amount of parameters.

**Comment by FauconFan** [comment]
I don't fix anything, because you are maybe working on it. and because that is your idea, and your baby. Tell me if you want some help ^^

**Comment by xavitator** [comment]
You need to write that we need to code in english in the [code of conduct](https://github.com/Heijix/nut-lang/blob/master/CODE_OF_CONDUCT.md) file.

**Comment by FauconFan** [comment]
updated ^^
